<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>258</width>
    <height>283</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>258</width>
    <height>283</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Nesicide Emulator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/resources/appicon.png</normaloff>:/resources/appicon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>256</width>
     <height>240</height>
    </size>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>257</width>
        <height>241</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: black;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>258</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEmulator">
    <property name="title">
     <string>Emulator</string>
    </property>
    <widget class="QMenu" name="menuSystem">
     <property name="title">
      <string>System</string>
     </property>
     <addaction name="actionNTSC"/>
     <addaction name="actionPAL"/>
     <addaction name="actionDendy"/>
    </widget>
    <widget class="QMenu" name="menuAudio">
     <property name="title">
      <string>Audio</string>
     </property>
     <addaction name="actionMute_All"/>
     <addaction name="separator"/>
     <addaction name="actionSquare_1"/>
     <addaction name="actionSquare_2"/>
     <addaction name="actionTriangle"/>
     <addaction name="actionNoise"/>
     <addaction name="actionDelta_Modulation"/>
    </widget>
    <addaction name="menuSystem"/>
    <addaction name="menuAudio"/>
    <addaction name="actionFullscreen"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEmulator"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Alt+Q</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionNTSC">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>NTSC</string>
   </property>
  </action>
  <action name="actionPAL">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>PAL</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About NESICIDE</string>
   </property>
  </action>
  <action name="actionController_Mapping">
   <property name="text">
    <string>Controller Mapping</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences...</string>
   </property>
  </action>
  <action name="actionMute_All">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Mute All</string>
   </property>
  </action>
  <action name="actionSquare_1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Square 1</string>
   </property>
  </action>
  <action name="actionSquare_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Square 2</string>
   </property>
  </action>
  <action name="actionTriangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Triangle</string>
   </property>
  </action>
  <action name="actionNoise">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Noise</string>
   </property>
  </action>
  <action name="actionDelta_Modulation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Delta Modulation</string>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fullscreen</string>
   </property>
   <property name="toolTip">
    <string>Enter fullscreen mode</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionDendy">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dendy</string>
   </property>
   <property name="toolTip">
    <string>Dendy</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
